### PR TITLE
COMMERCE-4792 Improve the naming of price-related entities and labels pt. 2

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:stop

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by marco-leo** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:stop

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-pricing-web/src/main/java/com/liferay/commerce/pricing/web/internal/frontend/CommercePromotionRuleCPDefinitionDataSetDataProvider.java [hunk 1]
```diff
@@ -90,11 +90,11 @@ public int getItemsCount(
 		return 0;
 	}
 
-	private List<DiscountRuleCPDefinition> _getDiscountRuleCPDefinitions(
+	private List<PromotionRuleCPDefinition> _getDiscountRuleCPDefinitions(
```
Comment (dsitu) [inline modules/apps/commerce/commerce-pricing-web/src/main/java/com/liferay/commerce/pricing/web/internal/frontend/CommercePromotionRuleCPDefinitionDataSetDataProvider.java:93]:
```suggestion
	private List<PromotionRuleCPDefinition> _getPromotionRuleCPDefinitions(
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Review by dsitu — APPROVED** [review]

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:stop

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Pull request closed without merging**